.github/workflows/release.yml

```diff
@@ -5,39 +5,46 @@ on:
       - prereleased
       - released
 
-env: 
- 
-  DOCKER_IMAGE_NAME: newrelic/k8s-webhook-cert-manager
-  DOCKER_IMAGE_TAG: ${{ github.event.release.tag_name }}
-
 jobs:
   publish:
-    name: Publish
+    name: Release docker images
     runs-on: ubuntu-latest
+    env:
+      DOCKER_IMAGE_NAME: roobre/k8s-webhook-cert-manager
+      DOCKER_PLATFORMS: "linux/amd64,linux/arm64,linux/arm"
     steps:
       - uses: actions/checkout@v2
 
-      - name: Extract version from release tag
+      - name: Generate docker image version from git tag
         run: |
-          echo "DOCKER_IMAGE_TAG=$(echo $DOCKER_IMAGE_TAG | grep -oE '([0-9.]*[0-9])$')" >> $GITHUB_ENV
+          echo "${{ github.event.release.tag_name }}" | grep -E '^v[0-9.]*[0-9]$'
+          DOCKER_IMAGE_TAG=$(echo "${{ github.event.release.tag_name }}" | sed 's/^v//')
+          echo "$DOCKER_IMAGE_TAG" | grep -E '^[0-9.]*[0-9]$'
+          echo "DOCKER_IMAGE_TAG=$DOCKER_IMAGE_TAG" >> $GITHUB_ENV
```

Comment:
isn't this a bit hackish?

Reply:
At some point when we drafted the pipelines we thought it was a good idea to do some validation.

Right now, I cannot remember what the intent behind the third line (`echo "$DOCKER_IMAGE_TAG" | grep -E '^[0-9.]*[0-9]$'`), since it is already validated 🤔

Reply:
ahhh first we check if it is in the v0.0.0 format, we strip the v, we check if it is in the 0.0.0 format and we overwrite DOCKER_IMAGE_TAG.

I do not like it that much, but no need to change it now

Reply:
I have removed the second check since it was a bit redundant.